miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -426,6 +426,25 @@ private static DetailedSample toDetailedSample(DetailedSampleDto from) {
     }
     to.setNonStandardAlias(from.getNonStandardAlias());
     to.setParent(getParent(from));
+
+    if (!LimsUtils.isStringEmptyOrNull(from.getExternalNames()) && to.getParent() != null) {
+
+      DetailedSample child = to;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "child". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)